Autopilot/TelemetryManager/Src/telemetryStateClasses.cpp

```diff
@@ -1,24 +1,34 @@
 /*
 * Author: Nixon Chan
-* Implementing Telem Comms: Gordon Fountain
+* Implementing Telem Comms: Gordon Fountain, Dhruv Rawat
 */
 
 #include "telemetryStateClasses.hpp"
 
+/***********************************************************************************************************************
+ * Definitions
+ **********************************************************************************************************************/
 
+Telemetry_PIGO_t obtainDataMode::_rawPMData; // CHANGE DATATYPE. ONLY USING THIS SO I CAN GET FLOW
+Telemetry_PIGO_t decodeDataMode::_decodedPMData;
 
-static Telemetry_PIGO_t _PMData;
-static Telemetry_POGI_t _GSData;
+Telemetry_POGI_t readFromPathMode::_rawGSData;
+Telemetry_POGI_t encodeDataMode::_encodedGSData; // CHANGE DATATYPE. ONLY USING THIS SO I CAN GET FLOW
+
+/***********************************************************************************************************************
+ * Code
+ **********************************************************************************************************************/
 
 void initialMode::execute(telemetryManager* telemetryMgr)
 {
-    //initial mode
-    //Initialize mail queue to send data to Path Manager:
-    TelemCommWithPMInit();
-    //Initializa mail queue to receive Path Manager data
+    //Initialize mail queue to SEND data to Path Manager
+    CommFromTMToPMInit();
+    
+    //Initialize mail queue to RECEIVE Path Manager data
+    CommFromPMToTMInit();
```

Comment:
This is redundant since each module initializes its own outgoing mail queue before sending data. If it is not already initialized, that means the other module has not yet sent any data, so GetFromPMToTM would return false either way.